Helper/Data.php

```diff
@@ -35,8 +35,8 @@ class Data extends AbstractHelper
     const TEST = 'test';
     const LIVE = 'live';
     // Only used for backend orders! Checkout in front-end is using different checkout version see web folder
```

Comment:
For the functions using it we have a deprecation comment so I think it makes sense to add it here as well
```suggestion
    // Only used for backend orders! Checkout in front-end is using different checkout version see web folder, will be removed in v8.0.0
```